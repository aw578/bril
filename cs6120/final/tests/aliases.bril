# constants dont propagate ...
@main {
  a: int = const 7;
  b: int = id a;
  c: int = id b;
  d: int = id a;
  e: int = id b;
}