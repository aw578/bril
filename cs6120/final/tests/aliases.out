Congruence classes for function main:
Class const 7: ['a.0']
Class id: ['d.0', 'b.0']
Class id_new: ['e.0', 'c.0']
